gpflow/inducing_variables/mo_inducing_variables.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018 GPflow authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .inducing_variables import InducingVariables
+
+
+class MultioutputInducingVariables(InducingVariables):
+    """
+    Multioutput Inducing Variables
+    Base class for methods which define a collection of inducing variables which
+    in some way can be grouped. The main example is where the inducing variables
+    consist of outputs of various independent GPs. This can be because our model
+    uses multiple independent GPs (SharedIndependentMok, SeparateIndependentMok)
+    or because it is constructed from independent GPs (eg IndependentLatentBase,
+    SeparateMixedMok).
+    """
+    pass
+
+
+class FallbackSharedIndependentInducingVariables(MultioutputInducingVariables):
+    """
```

Comment:
Should have the specific thing ("what is 'shared'") up front, not hidden at end of docstring.

Also, lots of redundancy with the Separate... class: can this be unified ?